static/htk/js/toolbar/emulate_user.js

```diff
@@ -27,51 +35,41 @@ $(function() {
         }
     }
 
-    function toggleContainers() {
-        if ($.cookie(EMULATE_USER_ID)) {
-            emulateUsernameContainer.hide();
-        } else if ($.cookie(EMULATE_USERNAME)) {
-            emulateUserIDContainer.hide();
-        } else {
-            emulateUserIDContainer.show();
-            emulateUsernameContainer.show();
-        }
-    }
-
-    function toggleForm(formName) {
-        var form = $('.emulate-' + formName + '-form');
-        var stopButton = $('.emulate-' + formName + '-stop');
-        var cookieName = getCookieName(formName);
-        if ($.cookie(cookieName)) {
-            form.hide();
-            stopButton.show();
+    function toggleForm() {
+        var activeCookie = getActiveCookie();
+        if (activeCookie) {
+            emulateUserForm.hide();
+            emulateUserStopButton.show();
         } else  {
-            form.show();
-            stopButton.hide();
+            emulateUserForm.show();
+            emulateUserStopButton.hide();
         }
     }
 
-    function get_expire_date() {
-        var date = new Date();
-        date.setTime(date.getTime() + HTK_EMULATE_USER_COOKIE_EXPIRE_TIME_SECONDS);
-        return date;
+    function showError() {
+        emulateUserMessage.html('Invalid Username or User ID');
+        emulateUserMessage.show();
     }
 
-    function getCookieName(name) {
-        var cookieNameParts = name.split('-');
-        var cookieName = 'emulate_user_' + cookieNameParts[cookieNameParts.length - 1];
-        return cookieName;
-    }
+    function handleEmulateUserButtonClicked(cookieName) {
+        var cookieVal = emulateUserInput.val();
+        if (cookieVal === '' || (cookieName === EMULATE_USER_ID && isNaN(cookieVal))) {
+            showError();
+            return;
```

Comment:
don't like early `return` statements.

put the rest of the function inside of `else`